define/build.go

```diff
@@ -120,7 +121,7 @@ type BuildOptions struct {
 	ContextDirectory string
 	// PullPolicy controls whether or not we pull images.  It should be one
 	// of PullIfMissing, PullAlways, PullIfNewer, or PullNever.
-	PullPolicy PullPolicy
+	PullPolicy config.PullPolicy
```

Comment:
This changes the default (unset) value from `PullIfMissing` to `config.PullPolicyAlways`.